### ISSUE TITLE
Ord browser companion 

### ISSUE THREAD

**Issue by elocremarc** [ordinals/ord#3790]
Ord is still the most cypherpunk wallet where you run everything and don't reuse addresses. However currently we have no way to interact with the browser ecosystem across ordinals. It would be nice if there was a companion browser extension.

**Comment by cryptoni9n** [comment]
I agree and was daydreaming of something similar recently.  It would be great if we could somehow extend ord wallet into a browser extension of its own.

**Comment by stet** [comment]
some prior discussion/art on this might be useful for reference.

https://github.com/tyjvazum/arb/issues/13

https://github.com/tyjvazum/arb-companion

**Comment by raphjaph** [comment]
Yeah, this would be cool!
